modules/ossm-sidecar-injection.adoc

```diff
@@ -13,3 +13,25 @@ The upstream version of Istio injects the sidecar by default if you have labeled
 
 The webhook checks the configuration of pods deploying into all projects to see if they are opting in to injection with the appropriate annotation.
 ====
+
+[id="ossm-sidecar-injection-env-var_{context}"]
```

Comment:
@knrc Can you take a look? This is my start to https://issues.redhat.com/browse/MAISTRA-1642.